suite/tests/CMakeLists.txt

```diff
@@ -4266,11 +4266,12 @@ if (BUILD_CLIENTS)
   endif ()
 
   if (proc_supports_pt)
-    if (BUILD_PT_TRACER AND BUILD_PT_POST_PROCESSOR)
+    # TODO i#5505: Due to the excessive warning messages generated by the decoding of
+    # invalid instructions during the execution of drcachesim, the regex matching function
```

Comment:
How about we print only the first ~10 instances of those error messages?

Reply:
Done.